nifi-nar-bundles/nifi-azure-bundle/nifi-azure-processors/src/test/java/org/apache/nifi/processors/azure/storage/AbstractAzureBlobStorage_v12IT.java

```diff
@@ -128,6 +144,26 @@ protected BlobClient uploadBlob(String blobName, byte[] blobData) throws Excepti
         return blobClient;
     }
 
+    protected BlobClient uploadBlobWithCSE(String blobName, byte[] blobData, String hexKey, String keyId, String keyWrapAlgorithm) throws Exception {
```

Comment:
The Integration Tests are not run automatically, so I recommend avoiding adding new integration tests for these changes.

Reply:
I would suggest to keep them as they helped me in identifying that KeyWrapAlgorithm cannot be constant and needs to derived from key size. Also helped me in verifying different key algorithm sizes